components/BottomSheet/src/MDCBottomSheetTransitionController.h

```diff
@@ -45,6 +45,11 @@
  */
 @property(nonatomic, assign) BOOL dismissOnBackgroundTap;
 
+/**
+ The height of the content view.
```

Comment:
Please expand on this documentation as well. I believe this is a pass-through property to the presentation controller and so (if set) will affect the initial height of the sheet when presented.

Reply:
Done